django-verdant/rca/templatetags/rca_tags.py

```diff
@@ -746,3 +749,17 @@ def double_click_ads(context):
         'cat': cat,
         'page': page
     }
+
+
+@register.filter(name='json_script')
+def json_script(value, element_id):
```

Comment:
Django has a `json_script` filter, is it not on the verdant Django version?

Reply:
Yea, it's basically a port of that because it's not in our version